units/XNA0303/XNA0303_unit.bp

```diff
@@ -319,14 +320,14 @@ UnitBlueprint {
                 },
             },
             RackFireTogether = false,
-            RackRecoilDistance = -1,
+            RackRecoilDistance = 0,
             RackReloadTimeout = 10,
             RackSalvoChargeTime = 0,
             RackSalvoReloadTime = 0,
             RackSalvoSize = 1,
             RackSlavedToTurret = false,
             RangeCategory = 'UWRC_AntiAir',
-            RateOfFire = 1.1,
+            RateOfFire = 1.43,
```

Comment:
other ASF has a rate of fire of 1, nomads had it at 1.1 to keep the stats pretty similar, is there a reason why it should be higher than the other factions? More projectiles in asf fights leads to more lag so its best to not have it higher if it doesnt specifically solve anything.

Reply:
Alternatively it might just be better to change the RoF to 1 and damage to 400 or 405 to match other ASF? Current values are RoF at 1.1 and Damage at 360, which both don't match other factions.

Reply:
Oh, that actually could be th reason why you even made it here. I suggest not experiment with damage and ROF for ASF and keep it standard at 400/405 damage per shot with 1 ROF